lib/wickedbase.pm

```diff
@@ -1109,6 +1130,43 @@ sub check_coredump {
     }
 }
 
+sub container_runtime {
```

Comment:
@jlausuch , @pdostal , @mloviska , @ricardobranco777 , @grisu48  -  WDYT about this ? is there chance to use more our existing codebase for containers ? 

In short : wicked tests need to use some utility . it was decided to wrap it into container . So we not testing containers here just using container as a tool for testing ... But I still have doubts if this functions should be in wickedbase and that we not duplicating logic which already exists in containers tests

Reply:
At a first glance, I don't have any objection keeping what Clemens has done here. It's very specific for this use case and he's already using functions from container libraries (install_docker, etc). Moving this to container libraries would require some small re-work in those libraries, not sure if it makes sense.

Reply:
ok , than let's keep it as "implementer" have chosen